CONTRIBUTING.md

```diff
@@ -129,8 +128,23 @@ git sparse-checkout add /common
 # etc
 ```
 
+
+## Commit Message Guidelines
+
+We follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) for commit messages.
+
+EG: `feat(name of feature): description of feature`
+
+## Pull Request Guidelines
+
+We follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) for pull request titles.
+
+EG: `feat(name of feature): description of feature`
+
```

Comment:
This isn't true, we don't enforce/follow any commit title convention, it's up to preference.

Reply:
Yeah true.
That an proposal to follow conventional commit